.changeset/many-wolves-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@balancer/sdk": minor
+---
+
+Adding Recovery Remove Liquidity for Balancer V2 Composable Stable and Weighted Pool types
```

Comment:
I just merged a PR with this in place. Make sure it's not duplicated before merging this one.